### ISSUE TITLE
`gpt-4-1106-preview`missing from LLM enums

### ISSUE THREAD

**Issue by kgrofelnik** [homanp/superagent-py#8]
I've created an agent using the admin interface and tried using it with the SDK.

It seems that `gpt-4-1106-preview` is missing.

```
Traceback (most recent call last):
  File "main.py", line 5, in <module>
    agent = client.agent.get("f60830db-8cd2-47c3-98ea-671ef4e0ee4c")
  File ".../lib/python3.10/site-packages/superagent/resources/agent/client.py", line 89, in get
    return pydantic.parse_obj_as(AppModelsResponseAgent, _response.json())  # type: ignore
  File ".../lib/python3.10/site-packages/pydantic/v1/tools.py", line 38, in parse_obj_as
    return model_type(__root__=obj).__root__
  File ".../lib/python3.10/site-packages/pydantic/v1/main.py", line 341, in __init__
    raise validation_error
pydantic.v1.error_wrappers.ValidationError: 2 validation errors for ParsingModel[AppModelsResponseAgent]
__root__ -> data -> llms -> 0 -> llm -> options
  str type expected (type=type_error.str)
__root__ -> data -> llmModel
  value is not a valid enumeration member; permitted: 'GPT_3_5_TURBO_16K_0613', 'GPT_3_5_TURBO_0613', 'GPT_4_0613', 'GPT_4_32K_0613' (type=type_error.enum; enum_values=[<LlmModel.GPT_3_5_TURBO_16_K_0613: 'GPT_3_5_TURBO_16K_0613'>, <LlmModel.GPT_3_5_TURBO_0613: 'GPT_3_5_TURBO_0613'>, <LlmModel.GPT_4_0613: 'GPT_4_0613'>, <LlmModel.GPT_4_32_K_0613: 'GPT_4_32K_0613'>])
```

**Comment by homanp** [comment]
@kgrofelnik thank you for reporting! Will fix!

**Comment by homanp** [comment]
@kgrofelnik this is now fixed in `v0.1.37`
